modules/nf-core/bamclipper/tests/main.nf.test

```diff
@@ -0,0 +1,65 @@
+
+nextflow_process {
+
+    name "Test Process BAMCLIPPER"
+    script "../main.nf"
+    process "BAMCLIPPER"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "bamclipper"
+
+    test("TruSight-Myeloid - [BAM, BEDPE]") {
+
+        when {
+            process {
+
+                """
+                input[0] = [
+                    [ id:'SRR2075598' ],
+                    file(params.test_data['sarscov2']['illumina']['test_paired_end_sorted_bam'], checkIfExists: true),
+                    file(params.test_data['sarscov2']['illumina']['test_paired_end_sorted_bam_bai'], checkIfExists: true),
+                    file(params.test_data['sarscov2']['genome']['test_bed'], checkIfExists: true)
```

Comment:
This looks like a BED file. BAMclipper actually expects a BEDPE file oO

Reply:
See new main.nf.test